common/src/main/res/values/strings.xml

```diff
@@ -970,7 +970,7 @@
     <string name="widget_text_color_white">White</string>
     <string name="widgets">Widgets</string>
     <string name="zone_event_failure">Unable to send zone event to Home Assistant</string>
-    <string name="companion_app">Companion App</string>
+    <string name="companion_app">Companion app</string>
```

Comment:
Actually this should still be a Title case to match the user coming from settings where they click on `Companion App`

Reply:
This was changed to "Companion app" in the latest frontend (so 2023.8.next or 2023.9)

Reply:
ah didn't catch that, disregard :)